### DISCUSSION TITLE
Location of nnUNet_train.py

### DISCUSSION THREAD

**Discussion by paul-reiners** [MIC-DKFZ/nnUNet#598] [Q&A] [answered]
The following instructions are given in ["Example: 3D U-Net training on the Hippocampus dataset"](https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/training_example_Hippocampus.md):

> Now you can already start network training. This is how you train a 3d full resoltion U-Net on the Hippocampus dataset:
> 
> `nnUNet_train 3d_fullres nnUNetTrainerV2 4 0`

However, I don't see a file named _nnUNet_train.py_.

Am I missing something here?

**Comment by FabianIsensee** [comment, marked as answer]
Have a look at the setup.py. In there you will see what file and function is called with each of the nnU-Net commands

**Comment by paul-reiners** [comment]
I'm using

`nnUNet/nnunet/run/run_training.py`

I hope that's the right file.  Looking at _setup.py_ did not help me find a file named _nnUNet_train.py_.

**Reply by FabianIsensee** [reply]
https://github.com/MIC-DKFZ/nnUNet/blob/0201e59cff564d451dafbae03ec9a41660ada959/setup.py#L30

how about this? :-D It points directly at the file you mentioned
